addons/auth_totp/tests/test_totp.py

```diff
@@ -81,3 +83,61 @@ def test_totp_administration(self):
         self.start_tour('/web', 'totp_tour_setup', login='demo')
         self.start_tour('/web', 'totp_admin_disables', login='admin')
         self.start_tour('/', 'totp_login_disabled', login=None)
+
+    def _build_payload(self, params={}):
```

Comment:
Why id there a default value? In which situation would that be useful? Also / alternatively, why not use kwargs?